### ISSUE TITLE
Error: Connection refused

### ISSUE THREAD

**Issue by cquillen2003** [klaemo/docker-couchdb#37]
Hi,

I'm following the steps to install couchdb:2.0-dev and it appears to install fine.  The command line response is exactly as expected: the 3 nodes checkout ok and it says 'Time to hack!', etc.

However, when trying to access Fauxton at http://server-ip-address:5984/_utils OR the server root at http://server-ip-address:5984, I keep getting the following error:

ERR_CONNECTION_REFUSED

The server is hosted at Digital Ocean.  Any help will be greatly appreciated.

**Comment by klaemo** [comment]
Did you expose the port with `-p 5984:5984`? Is it maybe firewalled?

**Comment by cquillen2003** [comment]
Thank you for getting back to me.  I did.  I tried both of these options per the docs:

docker run -p 5984:5984 klaemo/couchdb:2.0-dev
docker run -i -t klaemo/couchdb:2.0-dev --admin=foo:bar

Both commands appear to work, but I keep getting the 'Connection refused' error.  I tried 4-5 times on Digital Ocean, spinning up a new VM each time and 1 time on Linode.

**Comment by cquillen2003** [comment]
Not sure if you changed anything in the last couple of days, but I got it working.  I had to use the command in your GitHub readme documentation, not the command on Docker Hub.

Either of these commands do work:

docker run -p 15984:15984 -p 25984:25984 -p 35984:35984 --name couchdb klaemo/couchdb:2.0-dev
docker run -p 15984:15984 klaemo/couchdb:2.0-dev

The Docker Hub docs say 5984:5984, so that's the problem I think.
